opm/io/eclipse/ESmry.hpp

```diff
@@ -48,6 +51,7 @@ class ESmry
     const std::string& get_unit(const std::string& name) const;
 
 private:
+    Opm::filesystem::path rootName;
```

Comment:
I know it was `fs::path` instance - but the `rootName` does *not* actually represent a file system entry; it is "First name" - and then the actual files generated by simulator will have full names `$rootName.UNRST, $rootName.RSM, $rootName.SMSPEC, .....` and so on. So - in conclusion I would suggest that `rootName` was just `std::string`